example-apps/my-nest-project/src/users/user-history.entity.ts

```diff
@@ -0,0 +1,16 @@
+import {
+  History,
+  HistoryActionKind,
+  HistoryColumn,
+} from '@kittgen/nestjs-typeorm-history';
+import { Column, Entity } from 'typeorm';
+import { User } from './user.entity';
+
+@Entity()
+export class UserHistory extends User implements History {
+  @Column()
+  entityId: string;
+
+  @HistoryColumn()
+  action: HistoryActionKind;
+}
```

Comment:
how can user land add additional fields like:

- version number
- createdBy
- createdAt

Reply:
As discussed:
* above fields are properties of the entity - define mapping if these should be saved a properties of the history 
* different non-related usecase: support `JoinTable`